blueocean-rest/src/main/java/io/jenkins/blueocean/rest/pageable/PagedResponse.java

```diff
@@ -28,6 +28,7 @@
 @InterceptorAnnotation(PagedResponse.Processor.class)
 public @interface PagedResponse {
     class Processor extends Interceptor {
+        private static final int DEFAULT_LIMIT=100;
```

Comment:
Hey Vivek, is this default configurable in any way? Would it make sense to allow people set a startup prop (with an upper and lower bound)?